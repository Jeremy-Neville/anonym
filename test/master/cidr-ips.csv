test
166.186.169.4
166.186.169.4
0.0.0.0/0
166.186.169.4
166.186.169.4/32
166.186.169.0/24
82e2:e662:f728:b4fa:0000:0000:3093:08a4
82e2:e662:f728:b4fa:0000:0000:3093:08a4
::/0
82e2:e662:f728:b4fa:0000:0000:3093:08a4
82e2:e662:f728:b4fa::3093:8a4/128
82e2:e662:f728:b4fa::/64
