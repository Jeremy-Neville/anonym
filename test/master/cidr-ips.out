Processing input/cidr-ips.csv
